### ISSUE TITLE
NFR: Helix 5 Volvo Trucks NA Release steps

### ISSUE THREAD

**Issue by alexiscoelho** [hlxsites/vg-volvotrucks-us#716]
### Context/Description

Required steps to release the new Helix 5 Volvo Trucks NA 

### Acceptance Criteria

Adobe AC:
@dkuntze:
- [ ] Release

**Non-functional AC:**
- [ ] Update the [new repo code](https://github.com/aemsites/volvotrucks-) with the changes from the [upstream repo](https://github.com/hlxsites/vg-volvotrucks-us)
- [ ] Detach the new repo from the upstream, a manual request needs to be request to GitHub support
- [ ] Search functionality needs to be connected with the proper dev endpoints (different from the current one, @netmanito knows)
- [ ] Test Search and the publish and unpublished workflows

### Out of scope

- [ ] Transfer issues from the [original repo](https://github.com/hlxsites/vg-volvotrucks-us) into the [new repo](https://github.com/aemsites/volvotrucks-na)
- [ ] Configure the new repo issues into the agile board

**Comment by BeckyMedlin** [comment]
@mollyaljenkins Please link the search tickets that must be completed first to this ticket. Thank you!
